### PR TITLE
feat(tocco-ui): table component

### PR DESCRIPTION
**Pull request by MethodenMann**
- basic functionality of table, no order yet
- tests
- integration in showcase

### CODE REVIEW

File: packages/tocco-ui/src/Table/Table.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    if (props.cellRenderer) {
+      return props.cellRenderer(field, record)
+    } else {
+      return (
+        <span>{field.value.toString()}</span>
+      )
+    }
+  }
+
+  const getLabel = c => {
```
Comment (MethodenMann) [inline packages/tocco-ui/src/Table/Table.js]: const getLabel = c => (c.label || c.value)

File: packages/tocco-ui/src/Table/Table.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      }
+      </tbody>
+    </table>
+  )
+}
+
+Table.propTypes = {
+  /**
+   * Specifies the columns that are displayed. An array of objects containing an optional label, the value which
+   * will be referenced on each record as well as a optional order number. Lower numbers are getting displayed first.
```
Comment (ottrob) [inline packages/tocco-ui/src/Table/Table.js]: as 'an' optional order number instead of as a optional

File: packages/tocco-ui/src/Table/Table.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      {
+        value: React.PropTypes.string.isRequired,
+        label: React.PropTypes.string,
+        order: React.PropTypes.int
+      }
+    )
+  ).isRequired,
+  /**
+   * Records that get listed in the table.
+   * A records should consist of attributes that contains a value and a type attribute.
```
Comment (ottrob) [inline packages/tocco-ui/src/Table/Table.js]: contain instead of contains

File: packages/tocco-ui/src/Table/example.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import Table from './'
+
+// real-import:import {Table} from 'tocco-ui'
+
+export default () => {
+  const columnDefinitions = [
+    {
+      label: 'Vorname',
```
Comment (ottrob) [inline packages/tocco-ui/src/Table/example.js]: I'd use english labels since this project is open source.

**Pull request merged**